offocus/src/java/org/jivesoftware/openfire/plugin/ofmeet/FocusPlugin.java

```diff
@@ -86,6 +87,18 @@ private void ensureFocusUser()
                 Log.error( "Unable to provision a 'focus' user.", e );
             }
         }
+
```

Comment:
I've created https://github.com/igniterealtime/ofmeet-openfire-plugin/issues/12 to track this particular change.